src/MediaCollections/Models/Concerns/PathUuidSafe.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\MediaLibrary\MediaCollections\Models\Concerns;
+
+use Illuminate\Support\Facades\Storage;
+use Spatie\MediaLibrary\MediaCollections\Models\Media;
+use Spatie\MediaLibrary\Support\PathGenerator\PathGeneratorFactory;
+
+trait PathUuidSafe
+{
+    public static function bootPathUuidSafe()
+    {
+        if (! config('media-library.path_uses_uuid', false)) {
+            return;
+        }
+
+        static::updating(function (Media $media) {
```

Comment:
Could you add a test for this behaviour?

Reply:
Yes, sure.